doc/installation/centos.rst

```diff
@@ -4,10 +4,180 @@
 CentOS Installation
 -------------------
 
+Step-by-Step installation on CentOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. index:: CentOS, Red Hat, RHEL
 
-There is a detailed Howto [#howto]_ for installing privacyIDEA with FreeRADIUS
-3 on CentOS 7 using a python virtual environment.
+In this chapter we describe a way to install privacyIDEA on CentOS 7 based on the
+installation via :ref:`pip_install`. It follows the
```

Comment:
":ref:" does not seem to lead anywhere.

Reply:
The sphinx-tool is quite good at resolving references. When i build the documentation, this reference leads to [Python package index](https://github.com/privacyidea/privacyidea/blob/master/doc/installation/pip.rst) page.